Add files for ideapad slim 5x, tweaks to README.md

richardnpaul opened the pull request:
Feel free to push back on all the readme changes.  But the repo that is linked to doesn't have any releases so that needed changing and I had to check if the bin files were allcaps or just partial caps and it seems like it may be that they start with caps include but if they're all lowercase then they're not to be parsed.

alexVinarskis commented:
Thanks! indeed, I built and uploaded the tool to release of _this_ repo ;)

Could you please update README with this in mind, and also split README and Yoga change to different commits? Otherwise ill happily do it myself, however in that case the authorship of one of your commits will change

alexVinarskis commented:
Hmm are these all the files? Was hoping to see `qcdxkmext8380...` ones like in eg. [Zenbook](https://github.com/alexVinarskis/qcom-aeob-dumps/tree/master/asus-zenbook-a14), useful in context of the display.

richardnpaul commented:
> Thanks! indeed, I built and uploaded the tool to release of _this_ repo ;)

Okay, that is not what I understood from reading the README

> Could you please update README with this in mind, and also split README and Yoga change to different commits? Otherwise ill happily do it myself, however in that case the authorship of one of your commits will change

I can try to get that done tomorrow (for me).

alexVinarskis commented:
> Ahh those are not capitalised so I left them out. I can add them in.

Then your fix to readme is correct ;) Seems I forgot to update docs when I have discovered the new files

Thanks! Ill try to take a look tomorrow to see if I find anything useful wrt to getting Yoga to boot.

richardnpaul commented:
Okay, I have updated the PR, the readme commit is split out and I've updated it.  Give it another look over because I've changed it with my current understanding.  Also, I've converted all the bin files to json and added all of them in the second commit.  I've also rebased to pull in the licence.

alexVinarskis commented:
Thanks, LGTM!
I see there are now `CAM..` files, can you confirm those indeed do not exist for you?
